meson_options.txt

```diff
@@ -1,6 +1,7 @@
 option('build', type : 'combo', choices : ['all', 'standalone', 'library'], value : 'all', description : 'build type')
 option('consolekit', type : 'feature', description : 'ConsoleKit support', deprecated: {'true': 'enabled', 'false': 'disabled'})
 option('static_analysis', type : 'boolean', value : false, description : 'enable GCC static analysis support')
+option('gresource_quirks', type : 'boolean', value : false, description : 'compile the quirk files into the fwupd binary')
```

Comment:
I think this could be a good candidate for a feature.  Given it is only to be enabled on newer meson it can auto-disable on older etc.

Reply:
Done.